src/openrct2-ui/windows/TitleEditor.cpp

```diff
@@ -1,4 +1,4 @@
-/*****************************************************************************
+﻿/*****************************************************************************
```

Comment:
BOM?

Reply:
I have no idea why clang format is messing with me like this.